core/db/curibio/alembic/versions/da5a93c1bb08_update_uploadtype_and_scope_schema.py

```diff
@@ -58,9 +63,12 @@ def downgrade():
     op.execute("""CREATE TYPE "UploadType" AS ENUM('mantarray', 'nautilus', 'pulse2d')""")
     # set enums to column
     op.execute("""ALTER TABLE uploads ALTER COLUMN type TYPE "UploadType" USING type::text::"UploadType" """)
-    op.execute(
-        """ALTER TABLE jobs_result ALTER COLUMN type TYPE "UploadType" USING type::text::"UploadType" """
-    )
+
     # drop the old type, cleanup
     op.execute("DROP TYPE oldUploadType")
     op.execute("UPDATE uploads SET type='mantarray' where type='nautilus'")
+
+    op.execute("REVOKE ALL PRIVILEGES ON TABLE jobs_result FROM curibio_users")
```

Comment:
these should probably only revoke the `select` privileges

Reply:
ah, thanks